hr_expense_analytic_distribution/models/hr_expense.py

```diff
@@ -0,0 +1,130 @@
+# Copyright (C) 2021 Open Source Integrators
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class HrExpense(models.Model):
+
+    _inherit = "hr.expense"
+
+    def _get_account_move_line_values(self):
```

Comment:
The right way to do this is to call super() and get the result.
Then work on that data structure, copying and modifying the line data to perform the distribution.
Not using super() breaks the inheritance chain.

Reply:
I would love to see this implemented in under 40-50 LoC